extensions-core/s3-extensions/src/main/java/org/apache/druid/data/input/s3/S3InputSourceConfig.java

```diff
@@ -90,6 +91,24 @@ public boolean isCredentialsConfigured()
            secretAccessKey != null;
   }
 
+  @JsonIgnore
+  public boolean isAssumeRoleArnConfigured()
+  {
+    return assumeRoleArn != null && !assumeRoleArn.trim().isEmpty();
+  }
+
+  @JsonIgnore
+  public boolean isAssumeRoleArnEnvConfigured()
+  {
+    return !System.getenv(SDKGlobalConfiguration.AWS_ROLE_ARN_ENV_VAR).trim().isEmpty();
+  }
+
+  @JsonIgnore
+  public boolean isWebIdentityTokenEnvConfigured()
+  {
+    return !System.getenv(SDKGlobalConfiguration.AWS_WEB_IDENTITY_ENV_VAR).trim().isEmpty();
```

Comment:
NPE here to ?

Reply:
Don't `SDKGlobalConfiguration.AWS_ROLE_ARN_ENV_VAR` and `System.getenv` always return a string no matter what?

Reply:
@cryptoe addressed.